install/installer/pkg/components/server/configmap.go

```diff
@@ -237,6 +237,22 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 			// default limit for all cloneURLs
 			"*": 50,
 		},
+		WorkspaceClasses: []WorkspaceClass{
```

Comment:
Is this the config we want to deliver for Self-Hosted? :thinking:

Reply:
Good question. @kylos101 @atduarte Are we ok with these workspace classes for self-hosted?

Reply:
Don't think we would want to have the two workspace classes as default for Self-Hosted. It's an advanced feature and the initial installation should be as simple as possible imho. 

Would it be hard to have the default for self-hosted only be the standard?

cc @lucasvaltl as he is responsible for defining the self-hosted offering

Reply:
I think the idea was to deliver what we currently do for SH. This config should then be exposed under `experimental.webapp.workspaceClasses`, and the concrete values set [here](https://github.com/gitpod-io/ops/blob/main/deploy/production/meta-eu02/app/installer-config.yaml#L124).

Reply:
Yeah default for self-hosted for now should be a single workspace class - to keep installation simple. Right now even that class is not made obvious - but is rather just a default that sets a request but no limits. 

We should however think about how to properly allow for configuration of this (and https://github.com/gitpod-io/gitpod/issues/10805 is likely a better place to discuss this). Fwiw, I don't think it makes sense to make this part of the installation config - this might be something a user wants to change over time, specifically a user that is not the ["installer persona"](https://www.notion.so/gitpod/Persona-v0-2-Installer-c1329ac36ae74bf191f3d0d0a2b43756) but rather an ["project configurer"](https://www.notion.so/gitpod/Persona-v0-2-Project-Configurer-ffef632ab79a45cc8188c7cf7ffafd3d) / admin.